tools/autodeployment/env.example

```diff
@@ -16,3 +16,14 @@ UNSPLASH_CLIENT_ID=
 
 # Path to certificates
 PATH_TO_CERTS=
+
+# Slack "Incoming WebHook" to send messages to #telescope channel.
+# For more information: https://api.slack.com/messaging/webhooks
+# Example:
+# await fetch(SLACK_SEND_MESSAGE, {
+#       method: 'post',
+#       headers: { 'Content-Type': 'application/json' },
+#       body: "text": "This is the message"
+#     });
+# This will send the following message to our Slack channel: "This is the message"
+SLACK_SEND_MESSAGE=
```

Comment:
I'm not sure of what this means. Is `SLACK_SEND_MESSAGE` the text that will be sent to the channel?
Also, I don't think you need to show an example here, the URL to webhooks should be enough.

Reply:
Example requested by @humphd:
https://github.com/Seneca-CDOT/telescope/pull/1472#discussion_r533473388

`SLACK_SEND_MESSAGE` is an URL that receives a fetch/post with the message as the body and publishes it on our slack channel. It's almost like an API KEY, everyone with this URL will be able to send messages to our slack channel.

Reply:
I still think the example should go [in the docs](https://github.com/Seneca-CDOT/telescope/blob/master/docs/staging-production-deployment.md), where all the `env` vars are explained.

Reply:
Sure, you are right. We should add them to this doc too.

Reply:
If this is a URL, let's label it as such.  I think @manekenpix's confusion is partly due to the name.  What about `SLACK_WEBHOOK_URL` instead?